src/utils/cellSetOperations.js

```diff
@@ -68,10 +64,14 @@ const complement = (listOfSets, properties) => {
     (acc, curr) => new Set([...acc, ...curr]),
   );
 
+  const filteredCellIds = hierarchy.filter((rootCluster) => rootCluster.key === 'louvain')[0]
```

Comment:
you could replace
`hierarchy.filter((rootCluster) => rootCluster.key === 'louvain')[0]`
with
`hierarchy.find((rootCluster) => rootCluster.key === 'louvain')`

or perhaps:
`_.find(hierarchy, {key: 'louvain'})`